typescript/packages/plugins/allora/README.md

```diff
@@ -0,0 +1,38 @@
+# Goat Allora Plugin 🐐 (TypeScript)
```

Comment:
We should add to [their examples docs](https://github.com/goat-sdk/goat/tree/main/typescript/examples/vercel-ai/polymarket) too.

Reply:
Agreed! That'd be great! It will also allow you to test the plugin :)

Reply:
Done, works